msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -93,10 +102,19 @@ public abstract partial class ClientApplicationBase
 
             RequestContext requestContext = new RequestContext(new MsalLogger(Guid.Empty, null));
 
+            var platformInformation = new PlatformInformation();
```

Comment:
Can't you use the instance created at line 91?

Reply:
fixed